docs/reference/run.md

```diff
@@ -836,7 +836,7 @@ The container has unlimited memory which can cause the host to run out memory
 and require killing system processes to free memory. The `--oom-score-adj`
 parameter can be changed to select the priority of which containers will
 be killed when the system is out of memory, with negative scores making them
-less likely to be killed an positive more likely.
+less likely to be killed a positive more likely.
```

Comment:
can you add a "," before the "a" just so its easier to parse - since you're in here anyway.